### PR TITLE
FIX move admin_themes in YAML (#915)

### PR DESCRIPTION
**Pull request by ntd**
Naive fix for #915.

### CODE REVIEW

File: _config/forms.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 ---
 Name: adminforms
 ---
+SilverStripe\Admin\LeftAndMain:
+  admin_themes:
+    - 'silverstripe/admin:cms-forms'
+    - '$default'
+
```
Comment (michalkleiner) [inline _config/forms.yml]: Please move this block into a new file `_config/themes.yml` and give it `Name: adminthemes`, thanks.

**Review by michalkleiner — CHANGES_REQUESTED** [review]

File: _config/forms.yml (hunk 1 above)
Comment (ntd) [reply _config/forms.yml]: :heavy_check_mark:

**Review by michalkleiner — APPROVED** [review]
Looks ok to me, what do you think @kinglozzer?

**Comment by michalkleiner** [comment]
Any objections here @GuySartorelli?

File: code/LeftAndMain.php [hunk 2]
```diff
@@ -187,10 +187,7 @@ class LeftAndMain extends Controller implements PermissionProvider
      * @config
      * @var array
      */
-    private static $admin_themes = [
-        'silverstripe/admin:cms-forms',
-        SSViewer::DEFAULT_THEME,
-    ];
+    private static $admin_themes;
```
Comment (ChloeSartorelli) [inline code/LeftAndMain.php]: Add a note in the PHPDoc that this is set in `_config/themes.yml` - else it just looks like we don't have a default value here.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
I can't think of any way this would cause problems - seems like just a good change to make.

File: code/LeftAndMain.php (hunk 2 above)
Comment (ChloeSartorelli) [reply code/LeftAndMain.php]: Probably also assign an empty array by default
```suggestion
    private static $admin_themes = [];
```

Comment (michalkleiner) [reply code/LeftAndMain.php]: Done @GuySartorelli

**Review by GuySartorelli — APPROVED** [review]
Looks good. Merge on green.

**Pull request merged**